File: Genesis/Prosegur.Genesis.LogeoEntidades/Log/Movimientos/Movimiento.cs
using System;
using System.Collections.Generic;

namespace Prosegur.Genesis.LogeoEntidades.Log.Movimiento
{
    public class Movimiento
    {
        public string Identificador { get; set; }
        public string CodigoPais { get; set; }
        public string ActualID { get; set; }
        public DateTime FechaHoraPrimeraLlamada { get; set; }
        public DateTime FechaHoraUltimaLlamada { get; set; }
        public List<Llamada> Llamadas { get; set; }
        public Movimiento() 
        {
            Llamadas = new List<Llamada>();
        }
    }
}

File: Genesis/Prosegur.Genesis.Comon/Clases/Infraestructura/Log/Movimiento.vb
Imports Prosegur.Genesis.Comon
Namespace Infraestructura.Log
    Public Class Movimiento
        Public Property CodigoPais As String
        Public Property ActualID As String
        Public Property FechaHoraPrimeraLlamada As DateTime
        Public Property FechaHoraUltimaLlamada As DateTime
        Public Property Llamadas As List(Of Llamada)
        Public Sub New()
            Llamadas = New List(Of Llamada)()
        End Sub
    End Class
End Namespace
